monarchmoney/monarchmoney.py

```diff
@@ -276,10 +276,19 @@ async def get_subscription_details(self) -> Dict[str, Any]:
 
   async def get_transactions(
       self,
-      offset: int=0,
+      offset: Optional[int]=0,
       limit: int=1000,
       start_date: Optional[str]=None,
       end_date: Optional[str]=None,
+      search: str="",
+      categories: List[str]=[],
+      accounts: List[str]=[],
+      tags: List[str]=[],
```

Comment:
Done. The type should actually be `List[str]` since that is what the API returns and accepts. Using `List[int]` causes an error. For example, the category id is a numeric string that is 19 digits (i.e. "123456789012345678")

Reply:
Great, thank you for confirming.  I never know if a group's GraphQL will want `int` or `str`.  Maybe one day Monarch will let us pull the schema.